### DISCUSSION TITLE
http://localhost:8080/

### DISCUSSION THREAD

**Discussion by giscus** [Elijer/garden#21] [Announcements]
# http://localhost:8080/

Hi, I’m Elijah! 📸 by Jose Oliva Martinez @thatonejackedmexican I’m a programmer and artist from Arlington Virginia, currently based in NYC. Ask me about living in Uganda for a year, creating a tinyhome in a van, or my experience being medicated for ADHD in Highschool.

http://localhost:8080/

**Comment by Elijer** [comment]
Testing 123
